### PR TITLE
Add 5.56 AP and 7.62 AP rounds

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Followup to #69060
Adds originally intended armor piercing rounds, add them into proper cans
#### Describe the solution
Values are picked similar to 7.62x39 ap and 30-08 ap values, no precise math

### CODE REVIEW

File: data/json/items/ammo/223.json [hunk 1]
```diff
@@ -127,6 +127,15 @@
     "ascii_picture": "556_incendiary",
     "extend": { "effects": [ "INCENDIARY" ] }
   },
+  {
+    "id": "556_ap",
+    "copy-from": "556",
+    "type": "AMMO",
+    "name": { "str_sp": "5.56 NATO M995" },
+    "description": "An M995 armor piercing round with dense tungsten core and sharp, paint in black tip.  Higher material penetration in exchange of lower overall damage.",
```
Comment (TheShadowFerret) [inline data/json/items/ammo/223.json:135]:
```suggestion
    "description": "An M995 armor-piercing round using a 52 gr. projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Higher material penetration in exchange of lower overall damage.",
```
The other descriptions have bullet weights in them 👉👈

File: data/json/items/ammo/223.json [hunk 2]
```diff
@@ -127,6 +127,15 @@
     "ascii_picture": "556_incendiary",
     "extend": { "effects": [ "INCENDIARY" ] }
   },
+  {
+    "id": "556_ap",
+    "copy-from": "556",
+    "type": "AMMO",
+    "name": { "str_sp": "5.56 NATO M995" },
+    "description": "An M995 armor-piercing round using a 52gr projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Higher material penetration in exchange of lower overall damage.",
```
Comment (ymber) [inline data/json/items/ammo/223.json:135]:
```suggestion
    "description": "An armor piercing 5.56x45mm M995 round using a 52gr projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Increased penetration over standard ball rounds at the cost of worse wounding potential in soft tissue.",
```
Damage is a game mechanic and shouldn't be used that way in an item description.

File: data/json/items/ammo/308.json [hunk 3]
```diff
@@ -93,6 +93,15 @@
     "ascii_picture": "762_51_incendiary",
     "extend": { "effects": [ "INCENDIARY" ] }
   },
+  {
+    "id": "762_51_ap",
+    "copy-from": "762_51",
+    "type": "AMMO",
+    "name": { "str_sp": "7.62x51mm NATO M993" },
+    "description": "A 7.62x51mm NATO round with a 126gr tungsten carbide core for improved armor penetration, in exchange of damage penalty.",
```
Comment (ymber) [inline data/json/items/ammo/308.json:101]:
```suggestion
    "description": "An armor piercing 7.62x51mm round with a 126gr, tungsten carbide core projectile.  It offers significantly better armor penetration than standard ball rounds at the cost of worse wounding potential in soft tissue.",
```
Ditto.

**Review by ymber — COMMENTED** [review]
Good to bring these back, just needs some string changes.

**Comment by tenmillimaster** [comment]
Tell me more about this .30-08

**Comment by GuardianDll** [comment]
@tenmillimaster 
https://github.com/CleverRaven/Cataclysm-DDA/blob/17c23c24ede224a5f8cc79aeca27092bd1fdca52/data/json/items/ammo/3006.json#L89-L97

File: data/json/items/ammo/223.json (hunk 2 above)
Comment (GuardianDll) [reply data/json/items/ammo/223.json:135]: [<>](https://en.wikipedia.org/wiki/Damage)

**Comment by tenmillimaster** [comment]
> @tenmillimaster 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/17c23c24ede224a5f8cc79aeca27092bd1fdca52/data/json/items/ammo/3006.json#L89-L97

Ah I'm just pulling your leg about your description. :P

File: data/json/items/ammo/223.json (hunk 2 above)
Comment (ymber) [reply data/json/items/ammo/223.json]: It is a valid word but in the context it looks like it refers to the item stat and we don't like doing that in descriptions.

**Pull request merged**